BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqTransaction.java

```diff
@@ -64,7 +86,30 @@ public String getDescription() {
     }
 
     @Override
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    @Override
     public String toString() {
         return "<Payment desc:'" + getDescription() + "'>";
     }
+
+    public PaymentService.PostRequest convertToRequest() {
```

Comment:
Disagree we have many post requests (posssibly for each service) and that would decrease readablialty